charts/hail-search/Chart.yaml

```diff
@@ -19,7 +19,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.6
+version: 0.2.7
```

Comment:
Yep!  That's correct.  This doesn't actually put the code into `tgg-helm` though, so it'd essentially be buried behind the seqr PR as soon as that happens.

Reply:
since we bump this version is part of the prod seqr release I think it would be safer not to bump this as part of this PR, and then when we do the seqr release we will bump the version and it will include these helm changes and the updated seqr code in one release